libbeat/publisher/queue/spool/spool_test.go

```diff
@@ -50,6 +51,11 @@ func init() {
 }
 
 func TestProduceConsumer(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		// TODO (2018-07-24): Test is failing on Windows.
+		t.Skip("https://github.com/elastic/beats/issues/7720")
```

Comment:
I think this has since been addressed in both master and 6.x.

Reply:
It was fixed in https://github.com/elastic/beats/pull/7910.

Reply:
Good catch, #7859 was already backported so I'll make that change here

Reply:
and you beat me too it, thanks!